lib/src/install.rs

```diff
@@ -158,6 +156,10 @@ pub(crate) struct InstallConfigOpts {
     #[clap(long)]
     #[serde(default)]
     pub(crate) generic_image: bool,
+
+    /// The stateroot name to use. Defaults to `default`.
+    #[clap(long, hide = true)]
```

Comment:
We should at least debate: any reason not to stabilize (i.e. not `hide = true`) this? This really I guess is just the question of whether we want to tie bootc to the ostree concept of "stateroot" as is basically forever.

I think it's unlikely to be valuable for us to try to do anything really different here, so I'd say stabilize.

Reply:
>question of whether we want to tie bootc to the ostree concept of "stateroot" 

I guess in a future where that doesn't exist anymore, this flag can become a simple no-op